boards/stm32h743zi/src/bin/telemetry.rs

```diff
@@ -0,0 +1,127 @@
+#![no_std]
+#![no_main]
+
+use defmt::*;
+use embassy_executor::Spawner;
+use embassy_net::{Ipv4Address, Ipv4Cidr, Stack, StackResources};
+use embassy_stm32::{
+    bind_interrupts,
+    eth::{self, generic_smi::GenericSMI, Ethernet, PacketQueue},
+    gpio::Pin,
+    peripherals::{self, ETH},
+    rcc,
+    rng::{self, Rng},
+    time::Hertz,
+    Config,
+};
+use embassy_time::{Duration, Timer};
+use hyped_boards_stm32h743zi::{
+    config::{BOARD_STATIC_ADDRESS, MQTT_BROKER_IP},
+    log::log,
+    tasks::{
+        mqtt_send_and_recv::{mqtt_recv_task, mqtt_send_task, net_task},
+        test_mqtt_tasks::{button_task, five_seconds_task},
+    },
+};
+
+use hyped_core::log_types::LogLevel;
+use rand_core::RngCore;
+use static_cell::StaticCell;
+use {defmt_rtt as _, panic_probe as _};
+
+bind_interrupts!(struct Irqs {
+    ETH => eth::InterruptHandler;
+    RNG => rng::InterruptHandler<peripherals::RNG>;
+});
+
+#[embassy_executor::main]
+async fn main(spawner: Spawner) -> ! {
+    let mut config = Config::default();
+    {
+        use embassy_stm32::rcc::*;
+        config.rcc.hse = Some(Hse {
+            freq: Hertz(8_000_000),
+            mode: HseMode::Bypass,
+        });
+        config.rcc.pll1 = Some(rcc::Pll {
+            source: PllSource::HSE,
+            prediv: PllPreDiv::DIV4,
+            mul: PllMul::MUL216,
+            divp: Some(rcc::PllDiv::DIV2),
+            divq: None,
+            divr: None,
+        });
+        config.rcc.ahb_pre = AHBPrescaler::DIV1;
```

Comment:
https://github.com/embassy-rs/embassy/blob/f319f1bc1b36cd6c7860391e49083ee64c079547/examples/stm32h7/src/bin/eth.rs#L33 ?? Not sure about each individual clock value